### PR TITLE
Feature/styling changes

### PR DESCRIPTION
**Pull request by majsa1**
Some changes based on feedback

### CODE REVIEW

File: packages/client/src/app/challenge/components/ChallengeCheckBox.vue [hunk 1]
```diff
@@ -64,13 +64,18 @@ export default {
     }
   },
   computed: {
-    getClass() {
+    getButtonClass() {
       return this.checked
         ? 'position-absolute top-50 text-tertiary'
```
Comment (NoNamer777) [inline packages/client/src/app/challenge/components/ChallengeCheckBox.vue:69]: You can also choose to only return the class that changes and put the other classes in the template, if they won't charge depending on the checked start state

File: packages/client/src/app/challenge/components/ChallengeCheckBox.vue [hunk 2]
```diff
@@ -64,13 +64,18 @@ export default {
...
         : 'position-absolute top-50 text-secondary';
     },
     getCheckmarks() {
       return this.checked ? '✔' : this.dayNumber < this.todayNumber ? '✘' : ' ';
+    },
+    getDayClass() {
+      return this.dayNumber === this.todayNumber
+        ? 'fw-bold text-primary d-flex justify-content-center'
```
Comment (NoNamer777) [inline packages/client/src/app/challenge/components/ChallengeCheckBox.vue:77]: > You can also choose to only return the class that changes and put the other classes in the template, if they won't charge depending on the checked start state

Same here

File: packages/client/src/app/challenge/components/ChallengeProgress.vue [hunk 3]
```diff
@@ -111,6 +112,9 @@ export default {
       return this.challengeDays[this.todayNumber - 1].earned
         ? 'btn btn-secondary'
         : 'btn btn-primary';
+    },
+    hideFromScreenReader() {
+      return this.isOwner ? 'false' : 'true';
```
Comment (NoNamer777) [inline packages/client/src/app/challenge/components/ChallengeProgress.vue:117]: Why not just return booleans here?

**Review by NoNamer777 — CHANGES_REQUESTED** [review]
Left some comments

File: packages/client/src/app/challenge/components/ChallengeCheckBox.vue (hunk 1 above)
Comment (majsa1) [reply packages/client/src/app/challenge/components/ChallengeCheckBox.vue]: Good point, adjusted

File: packages/client/src/app/challenge/components/ChallengeProgress.vue (hunk 3 above)
Comment (majsa1) [reply packages/client/src/app/challenge/components/ChallengeProgress.vue]: Haha, a little verbose this way, fixed!

**Review by NoNamer777 — APPROVED** [review]
LGTM 👌🏼

**Pull request merged**